src/utils/setup/create-video-pages.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+import { fetchBuildTimeMedia } from './fetch-build-time-media';
+import { Video } from '../../interfaces/video';
+
+export const createVideoPages = async (
+    createPage: Function,
+    metadata: object
+) => {
+    const { allVideos } = await fetchBuildTimeMedia();
+
+    allVideos.forEach((video: Video) => {
+        createPage({
```

Comment:
Oh nice! Easy enough to construct these pages. Would we like to do feedback on the pages themselves in this PR or should we come back to it after the PR?